requirements_test.txt

```
@@ -12,13 +12,15 @@ meshio<5.4.0
 nest_asyncio<1.5.9
 numpydoc==1.6.0
 param<2.1.0
+pydata-sphinx-theme==0.14.3
```

I think the convention here is to limit the dependency and only pin for specific, high-impact libraries like `numpydoc` or `pytest-pyvista`. That way dependabot will bump it on the next minor release. Unless you think we are doing something that where a patch release of this package could affect us?

```suggestion
pydata-sphinx-theme<0.15.0
```

---

Different files have different conventions for us:

- `requirements_docs.txt`: pin everything because our documentation build tends to be fragile and is only *fully* executed around releases
- `requirements_test.txt`: limit dependencies by minor release and let dependable bump these which we can easily pick up on errors, deprecation warnings, or breakages
- `requirements.txt`: same as `requirements_test.txt` but this is just the core dependencies

---

Thank you for the explanation ! I don't have any hard requirement here. I added the two packages to enable tabb-ed interface in `tinypages`.  I just copied the versions from `requirements_docs.txt` .

---

I added your nice description in the commit message for future reference

---

> I added your nice description in the commit message for future reference

I'm not sure that will be preserved since we have a policy of squashing and merging all commits. ...though github may do something fancy to combine te commit messages when squashing

---

I guess @ChristosT is trying to follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/). I love that mind. The commit message itself will remain in this PR, which is very helpful to us.